setup.cfg

```diff
@@ -39,13 +39,17 @@ zip_safe = True
 python_requires = >= 3.7
 
 [options.extras_require]
+backend-clickhouse =
+    clickhouse-connect[numpy,pandas]<0.6
+    python-dateutil>=2.8.2
 backend-es =
     elasticsearch>=8.0.0
 backend-ldp =
     ovh>=1.0.0
     requests>=2.0.0
 backend-mongo =
     pymongo[srv]>=4.0.0
+    python-dateutil>=2.8.2
```

Comment:
Sure this is required?

Reply:
Mongo uses this: https://github.com/openfun/ralph/blob/master/src/ralph/backends/database/mongo.py#L10 and it did throw an error for me at one point. I can remove it if you think it shouldn't be an issue.

Reply:
Wow, this is a regression from our part, thanks for having spotted this! :muscle: